src/vorta/views/archive_tab.py

```diff
@@ -248,6 +263,13 @@ def populate_from_profile(self):
             self.mount_points, repo_mount_points = get_mount_points(profile.repo.url)
             if repo_mount_points:
                 self.repo_mount_point = repo_mount_points[0]
+                mount_point_filename = os.path.basename(self.repo_mount_point)
+                if mount_point_filename.startswith("vorta-quick-mount-"):
+                    self.quick_mount = True
+            for mount_point in self.mount_points:
+                mount_point_filename = os.path.basename(self.mount_points[mount_point])
+                if mount_point_filename.startswith("vorta-quick-mount-"):
+                    self.quick_mount_points[mount_point] = self.mount_points[mount_point]
```

Comment:
How do you know which repo a mount point belongs to?

Reply:
If I trigger `bmountrepo_refresh` on load, I can handle mounting/unmounting different repos.